bool request
---
float32 xCoordinate
float32 yCoordinate
float32 zCoordinate